zhazam.xcodeproj/project.pbxproj

```diff
@@ -36,6 +38,8 @@
 		0D1F8A5A230C5FF500ECE3B4 /* Images.xcassets */ = {isa = PBXFileReference; lastKnownFileType = folder.assetcatalog; path = Images.xcassets; sourceTree = "<group>"; };
 		0D1F8A5F230C5FF500ECE3B4 /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
 		0DCC8FDE2358D1670047EB90 /* LoadingViewController.xib */ = {isa = PBXFileReference; lastKnownFileType = file.xib; path = LoadingViewController.xib; sourceTree = "<group>"; };
+		24A6174D244FA9CA007F92E6 /* StartGameTimerViewController.xib */ = {isa = PBXFileReference; lastKnownFileType = file.xib; path = StartGameTimerViewController.xib; sourceTree = "<group>"; };
+		24A6174F244FA9F8007F92E6 /* StartGameTimerViewController.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = StartGameTimerViewController.swift; sourceTree = "<group>"; };
```

Comment:
Try to name your files more clearly. It has to be brief and specific. CountdownViewController maybe?

Reply:
Ok, good naming:)